api_tests/licenses/views/test_license_list.py

```diff
@@ -1,41 +1,35 @@
-from nose.tools import *  # flake8: noqa
+import pytest
 
-from tests.base import ApiTestCase
-from osf.models.licenses import NodeLicense
 from api.base.settings.defaults import API_BASE
+from osf.models.licenses import NodeLicense
 
+@pytest.mark.django_db
+class TestLicenseList:
 
-class TestLicenseList(ApiTestCase):
-    def setUp(self):
-        super(TestLicenseList, self).setUp()
-        self.licenses = NodeLicense.find()
+    def test_license_list(self, app):
+        licenses = NodeLicense.find()
```

Comment:
.find() is another one of those old style modm methods. When called without query params it just calls `NodeLicense.objects.all()` -- you can just call that directly instead